packages/node_modules/@webex/internal-plugin-conversation/test/integration/spec/get.js

```diff
@@ -293,5 +293,115 @@ describe('plugin-conversation', function () {
             assert.equal(mentions[0].url, activity.url);
           })));
     });
+
+    // TODO: add testing for bulk_activities_fetch() with clusters later
+    describe('#bulk_activities_fetch()', () => {
```

Comment:
This needs to be updated to the `bulkActivitiesFetch` method name

Reply:
Sorry I did that on another commit was having trouble getting the tests to run